code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1794,9 +1788,12 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 
 /datum/species/proc/althelp(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
 	if(user == target && istype(user))
-		if(user.getStaminaLoss() >= STAMINA_SOFTCRIT)
+		if(IS_STAMCRIT(user))
 			to_chat(user, "<span class='warning'>You're too exhausted for that.</span>")
 			return
+		if(!IS_COMBAT_ACTIVE(user))
+			to_chat(user, "<span class='warning'>Your muscles need to be tensed to do that!</span>")
```

Comment:
I'm not sure if this is even possible for it to come up, but this seems kind of... non-descriptive? I can easily see how someone would be confused by this message if it ever came up.

Reply:
yeah alright and it comes up if you're combat mode locked but toggled on